src/intervals/rounding.jl

```diff
@@ -187,7 +248,9 @@ function make_interval(T, expr1, expr2)
 end
 
 
-float(x::Interval) = make_interval(Float64, x)
+float(x::Interval) = Interval(convert(Float64,x.lo),convert(Float64,x.hi))
+# float(x::Interval) =
+#     @controlled_round(Float64, convert(Float64, x.lo), convert(Float64, x.hi))
```

Comment:
I think this also disturbs me, which may be the reason I left that thing commented there, but when I was using it, some kind of problem/inconsistency with the ITF1788 suite was appearing...